react-native.config.js

```diff
@@ -1,4 +1,6 @@
+const extraNodeModules = require('node-libs-browser');
```

Comment:
I think this was from a previous implementation test, is it still required with this implementation?